src/Language/PureScript/Errors.hs

```diff
@@ -1190,11 +1190,13 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
     printRow f t = markCodeBox $ indent $ f prettyDepth t
 
     -- If both rows are not empty, print them as diffs
-    printRows :: Type a -> Type a -> (Box.Box, Box.Box)
-    printRows r1@RCons{} r2@RCons{} = let
+    -- If verbose print all rows else only print unique rows
+    printRows :: Bool -> Type a -> Type a -> (Box.Box, Box.Box)
+    printRows True r1@RCons{} r2@RCons{} = (printRow typeDiffAsBox r1, printRow typeDiffAsBox r2)
```

Comment:
I think this should be `(printRow typeAsBox r1, printRow typeAsBox r2)`; since we are not eliding any labels from the rows in this instance, we don't want to display an ellipsis (which is what `typeDiffAsBox` does).

Reply:
Yes, not sure what I was thinking that day... Or, I probably did not completely understand the function.. Thanks for the clarification!